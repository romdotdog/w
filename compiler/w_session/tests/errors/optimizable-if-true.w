fn main(a: u32, b: u32): u32 {
    if 1 {
        a
    } else {
        b
    }
}