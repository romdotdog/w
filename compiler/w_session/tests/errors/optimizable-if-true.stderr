tests/errors/optimizable-if-true.w:2:8: error: condition evaluates to true
    if 1 {
       ^
